specs/light_client/sync_protocol.md

```diff
@@ -62,12 +61,9 @@ class LightClientUpdate(container):
     committee_branch: Vector[Hash, PERSISTENT_COMMITTEE_ROOT_IN_BEACON_STATE_DEPTH + log_2(SHARD_COUNT)]
 ```
 
-## Helpers
-
 ### `LightClientMemory`
 
 ```python
-@dataclass
 class LightClientMemory(object):
```

Comment:
see https://github.com/ethereum/eth2.0-specs/pull/1316#issuecomment-521697986
it should not be SSZ container class. 😄

Reply:
Ahh ok. I'll revert the change. Thank you!